haicrypt/hcrypt_tx.c

```diff
@@ -69,7 +69,8 @@ int HaiCrypt_Tx_ManageKeys(HaiCrypt_Handle hhc, void *out_p[], size_t out_len_p[
 	/* Manage Key Material (refresh, announce, decommission) */
 	hcryptCtx_Tx_ManageKM(crypto);
 
-	if (NULL == (ctx = crypto->ctx)) {
+	ctx = crypto->ctx;
+	if (NULL == ctx) {
```

Comment:
Simpler `if (!ctx)` ("standard valid pointer check"), or - if you prefer - `if (ctx == NULL)`.  [suggestion]